DEVELOPMENT.md

```diff
@@ -0,0 +1,82 @@
+# Development Guide
+
+> **Note**
+> This assumes that you have already followed the steps in the [README.md](./README.md) to set-up your OpenShift Clusters, OpenDataHub, ACM, OpenShift Pipelines, etc.
+
+## Useful Information
+
+### Models/Apps
+
+The GitHub repository is split into two main parts, Pipelines and ACM. Working on one part usually doesn’t involve working with the other, so you usually won’t have to set-up both dev environments.
+
+One aspect that is similar across both are the *models*/*apps*:
+
+* *Models* (used by Pipelines): These are the actual trained model files generated by Python ([`tf2model/`](https://www.tensorflow.org/guide/checkpoint), [`model.pkl`](https://scikit-learn.org/stable/model_persistence.html), etc.)
+* *Apps* (used by ACM): TL;DR: Sets of `YAML` files that reference images built using Pipelines and uploaded to Quay. See [RHACM Managing Applications](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/applications/managing-applications).
+
+We are currently using two sets of these as examples:
+
+> **Note**
+>
+> 1. [Example TensorFlow model from the MLflow GitHub repository](https://github.com/mlflow/mlflow/tree/ef0f922ca874dd94f9b88e8bfbde2a8cb2c385d2/examples/tensorflow)
```

Comment:
Hmmm, actually, the `mlflow` repository does not contain the model (the `tf2model`). It contains merely the code to train the model.

Reply:
> Hmmm, actually, the `mlflow` repository does not contain the model (the `tf2model`). It contains merely the code to train the model.

Yep, `mlflow/examples/tensorflow/` and `opendatahub-ai-edge/pipelines/models/tensorflow-housing/` are very different. They might seem like copies of each other since they contain similar files e.g. `conda.yaml`, however they are not at all. The folder in the repo is a copy of the `artifacts/model` folder that gets generated when you train the model:
![image](https://github.com/opendatahub-io/ai-edge/assets/99439005/7a9543fb-8e53-4294-94c7-1c40291f5c49)